IoProtocol.h

```diff
@@ -504,22 +504,19 @@ typedef struct {
 
 //-----------------------------------------------------------------------------
 // Request for Multi IO modules:
-typedef struct {
-	//bitfield: mode
-	INT8U i8uCalibrationMode;
-	//channels to calibrate
-	INT8U i8uChannels;
-	//specifies point in lookupTable
-	INT8U i8uPoint;
-	INT16S i16sCalibrationValue;
-} SMioCalibrationRequestData;
+struct calibration_data {		// IOP_TYP1_CMD_DATA6
+	u8 mode;
+	u8 channel;
+	u8 refPoint;
+	s16 val;
+} __attribute__((__packed__));
 
 typedef struct {
 	// IOP_TYP5_CMD_DATA6
 	UIoProtocolHeader uHeader;
-	SMioCalibrationRequestData sData;			/*  5 bytes */
+	struct calibration_data sData;				/*  5 bytes */
 	INT8U  i8uCrc;
-} SMioCalibrationRequest;
+} __attribute__((__packed__)) SMioCalibrationRequest;
```

Comment:
Adding `__attribute__((__packed__))` isn't necessary here because of the `#pragma pack(push,1)` further up in this file.

Reply:
I would like to eventually get rid of the pragma directive, since it is much more verbose to explicitly mark a structure as being packed than to have that pragma somewhere at the beginning of the file which can easily been overseen. Also it is not immediately clear when the pragma ends and which structures and _not_ packed.